MapboxNavigation/RouteVoiceController.swift

```diff
@@ -159,6 +159,11 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
             return false
         }
         
+        guard let userDistance = notification.userInfo![RouteControllerAlertLevelDidChangeNotificationDistanceToEndOfManeuverKey] as? CLLocationDistance else {
+            assert(false)
```

Comment:
Instead of `guard`ing, only to fail hard if the condition evaluates to false, it would be clearer to force-unwrap the `userInfo` entry using `as!`.